src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -122,7 +122,11 @@ const AssigneeSelector = createReactClass({
     return AssigneeSelector.filterAssignees(
       (ProjectsStore.getAll().find(p => p.slug == group.project.slug) || {
         teams: [],
-      }).teams,
+      }).teams.sort((a, b) => {
+        if (a.slug > b.slug) return 1;
+        if (a.slug < b.slug) return -1;
+        return 0;
```

Comment:
is there a case where we will have duplicate slugs, and need to preserve order?
if not, we can simplify as sort(a, b) => a.slug > b.slug

Reply:
nope, that shouldn't ever happen because team slugs should be unique across org. i'll update

Reply:
I have some bad news – I think @macqueen had it right the first time. `Array.prototype.sort` expects a compare fn that returns -1, 0, or 1 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort).